taskcluster/kinds/inference/kind.yml

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - translations_taskgraph.transforms.worker_selection
+    - taskgraph.transforms.task_context
+    - taskgraph.transforms.run:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+
+task-defaults:
+  description: "Run tests"
+  task-context:
+    from-parameters:
+      base_rev: base_rev
+    substitution-fields:
+      - run.command
+  run:
+    using: run-task
+    cwd: '{checkout}'
+  worker:
+    # 128 happens when cloning this repository fails
+    retry-exit-status: [128]
+
+tasks:
+  build:
+    worker-type: b-cpu
+    worker:
+      max-run-time: 3600
+      docker-image: {in-tree: inference}
+    run:
+      command:
+        - bash
+        - -c
+        - >-
+            task inference-build
+
+  build-wasm:
+    worker-type: b-cpu
```

Comment:
If you want, you could move `worker-type` and `worker` up to `task-defaults` seeing as they're the same in all `tasks` entries.

Reply:
I've refactored these configurations into the `task-defaults` section.